apps/website/src/pages/docs/surface.mdx

```diff
@@ -9,23 +9,40 @@ group: core
 
 import PropsTable from '~/components/PropsTable.astro';
 import LiveExample from '~/components/LiveExample.astro';
-import Placeholder from '~/components/Placeholder.astro';
 import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-surface' />
-
 <LiveExample src='Surface.main.tsx' truncate={false}>
   <AllExamples.SurfaceMainExample client:load />
 </LiveExample>
 
-## Custom
+## Variants
+
+### Elevation
+
+Elevation is measured as the distance between surfaces.
+
+<LiveExample src='Surface.elevation.tsx' truncate={false}>
```

Comment:
Remove `truncate` prop to allow truncation on this example.

Reply:
Fixed